lib/middleware/karma.js

```diff
@@ -101,7 +101,19 @@ var createKarmaMiddleware = function(filesPromise, serveStaticFile,
     // or debug.html - execution context without channel to the server
     if (requestUrl === '/context.html' || requestUrl === '/debug.html') {
       return filesPromise.then(function(files) {
-        serveStaticFile(requestUrl, response, function(data) {
+
+        var fileServer;
+        if (client && client.contextFile) {
+            fileServer = serveFile;
```

Comment:
it seems like the rest of the file follows 2-space indentations; you have 4-space